inc/classes/optimization/CSS/class-minify.php

```diff
@@ -119,8 +119,11 @@ private function replace_url( $url ) {
 			return false;
 		}
 
+		// This filter is documented in /inc/classes/optimization/class-abstract-optimization.php.
```

Comment:
Let's move and rename this file for our new standards and architecture.

Reply:
Not sure why it's not showing, but it has been done